editor/import/3d/post_import_plugin_skeleton_rest_fixer.cpp

```diff
@@ -42,6 +42,7 @@ void PostImportPluginSkeletonRestFixer::get_internal_import_options(InternalImpo
 		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/apply_node_transforms"), true));
 		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/normalize_position_tracks"), true));
 		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/overwrite_axis"), true));
+		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/keep_unmapped_axis"), false));
```

Comment:
If such an option is added, it should only display when override rest is enabled.

Also, the naming must be changed since the bones between the mapped bones will be changed even though they are not mapped. Such as `Keep Global Rest Outside of Map` since global rest is kept but local rest is changed.

Reply:
Maybe `Skip Unmapped Ends`? Also, make it a subcategory with the title "Overwrite Axis" and options `Enabled` and `Skip Unmapped Ends`.

I'm not strong on compatibility either. I have been using `Overwrite Axis` disabled since 4.2 because I looks like a bug, but once we introduce characters with different body types, this option is unavoidable. I can't think of a common situation where a user wants these bones to reset rotation on import because their only purpose is to transform attachments.

Reply:
Does "Preserve leaf axes" sound better?

I think I am feeling a bit better about this after some thought.

The other thing is I would like this to default to true (don't overwrite leaf bone axis) on new imports if possible, but I am not sure how.

As long as the behavior does not change for existing .import files, I think I would be ok getting this in for 4.3